src/foam/nanos/measure/Measure.js

```diff
@@ -37,7 +37,7 @@ foam.CLASS({
       name: 'daoKey'
     },
     {
-      class: 'string',
+      class: 'String',
```

Comment:
How is this used, we have a TimeZone model:
    {
      class: 'Reference',
      of: 'foam.time.TimeZone',
      name: 'timeZone',
      value: 'GMT'
    },

Reply:
It just a timezone string that pass to the foam.util.DateUtil.getTimeZoneId. 
but, Yes, we also can use reference of foam.time.TimeZone as property type.